src/lib_rules.ml

```diff
@@ -395,6 +396,7 @@ module Gen (P : Install_rules.Params) = struct
         ~dir
         ~dir_kind
         ~obj_dir
+        ~private_obj_dir:private_obj_dir
```

Comment:
this will soon be formatted away, but you can remove the duplication here

Reply:
yeah, good call